README.md

```diff
@@ -1,4 +1,8 @@
 # EarlGrey
+[![Apache License](https://img.shields.io/badge/license-Apache%202-lightgrey.svg?style=flat)](https://github.com/google/EarlGrey/blob/master/LICENSE)
+[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
```

Comment:
That's a good point. Carthage is fully supported but only if you know how to manually configure the testing target which is a complex and painful process. I'll remove the Carthage badge for now.